src/projects.h

```diff
@@ -78,6 +79,14 @@ extern "C" {
 #  define ABS(x)        ((x<0) ? (-1*(x)) : x)
 #endif
 
+#if INT_MAX == 2147483647
+typedef int int32;
```

Comment:
Can we use some 'namespace' ? So pj_int32. I know a number of other projects, which can potentially be mixed with proh, that also have typedef int32...

Reply:
That makes sense. I will change it.